.github/workflows/main.yml

```diff
@@ -0,0 +1,57 @@
+name: Generate Interop Report
+
+on:
+  workflow_dispatch:
+  push:
+  schedule:
+    # update the integration suite once per week at Sunday 5am UTC
+    - cron: '0 5 * * 0'
+
+jobs:
+  lint:
+    if: ${{ github.event_name == 'push' }}
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [16.x]
```

Comment:
Run all the tooling on `20.x` if possible.

Reply:
Addressed here: https://github.com/digitalbazaar/vc-data-model-2-test-suite/pull/6/commits/2ad83142ae145d3dbc1b214cfa869cb074988d59